incydr/cli/cmds/cases.py

```diff
@@ -117,10 +256,212 @@ def bulk_update(ctx: Context, csv: Path):
+    """
+    client = ctx.obj()
+    client.cases.v1.download_file_for_event(case_number, event_id, path)
+
+
+@cases.group(cls=IncydrGroup)
+def file_events():
+    """View and update file events associated with a case."""
+    pass
+
+
+@file_events.command("show", cls=IncydrCommand)
+@click.argument("case_number")
+@click.argument("event_id")
+@single_format_option
+@click.pass_context
+def show_file_event(
+    ctx: Context, case_number: int, event_id: str, format_: SingleFormat
+):
+    """
+    Show details for a file event attached to a case.
+    """
+    client = ctx.obj()
+    event = client.cases.v1.get_file_event_detail(case_number, event_id)
+
+    if format_ == SingleFormat.rich and client.settings.use_rich:
+        render_file_event(event)
+
+    elif format_ == SingleFormat.json:
+        console.print_json(event.json())
+
+    else:
+        echo(event.json())
+
+
+@file_events.command("list", cls=IncydrCommand)
+@click.argument("case_number")
+@table_format_option
+@columns_option
+@click.pass_context
+def list_file_events(
+    ctx: Context, case_number: int, format_: TableFormat, columns: str
+):
+    """
+    List file events attached to a case.
+    """
+    client = ctx.obj()
+    events = client.cases.v1.get_file_events(case_number).dict()["events"]
+
+    output_response_format(
+        events,
+        f"Case {case_number}: File Events",
+        format_,
+        columns,
+        client.settings.use_rich,
+    )
+
+
+csv_option = click.option(
+    "--csv", is_flag=True, default=False, help="event IDs are specified in a CSV file."
+)
+
+
+@file_events.command(cls=IncydrCommand)
+@click.argument("case_number")
+@click.argument("event_ids")
+@csv_option
+@click.pass_context
+def add(ctx: Context, case_number: int, event_ids: Union[str, Path], csv: bool):
+    """
+    Attach file events to a case specified by CASE_NUMBER.
+
+    EVENT_IDS is a comma-delimited string of event IDs to add to the case:
+
+        add CASE_NUMBER EVENT_IDS
+
+    To read the event IDs from a csv (single column, no header),
+    pass the path to a csv along with the --csv flag:
+
+        add CASE_NUMBER CSV_PATH --csv
+    """
+    client = ctx.obj()
+    client.cases.v1.add_file_events_to_case(
+        case_number, _parse_event_ids(event_ids, csv)
+    )
+
+
+@file_events.command(cls=IncydrCommand)
+@click.argument("case_number")
+@click.argument("event_ids")
+@csv_option
+@click.pass_context
+def remove(ctx: Context, case_number: int, event_ids: str, csv: bool):
+    """
+    Remove file events from a case specified by CASE_NUMBER.
+
+    EVENT_IDS is a comma-delimited string of event IDs to add to the case:
+
+        add CASE_NUMBER EVENT_IDSs
+
+    To read the event IDs from a csv (single column, no header),
+    pass the path to a csv along with the --csv flag:
+
+        add CASE_NUMBER CSV_PATH --csv
```

Comment:
Added this `--csv` flag to allow users to read in alert IDs from a CSV file. Open to discussion if you think there's a more user-friendly way to do this 👀

Reply:
Yeah, I was just looking at this. I think it'd be great if a user could pipe the output of a `file-events search` right into a case, like: 
```
incydr file-events search --start P10D --username '<subject_of_case>' --format csv | incydr cases file-events add 23 - --format csv
```

And we can copy the [FileOrString](https://github.com/code42/code42cli/blob/main/src/code42cli/click_ext/types.py#L29) class from the code42cli to allow the arg to be a string, filename (with `@` prefix), or stdin. 

We could probably also support json as an input format too.

Reply:
For sure, for a case like that should we add another flag like `--use-headers` or something to indicate that the CSV is providing headers/there may be extra unnecessary columns?

Reply:
I think it'd be easiest to just always require headers for CSV inputs.

Reply:
Thats true, thats easy to do.
I'm more wondering if we want to enable people to pipe data like this the column naming might not always be consistent.  `file-events search` will provide id's in a column named `event.id`, but another might provide them as `eventId` or `event_id` for example, but I'm not sure what we could do about that

Reply:
Hmm, good question. I suppose we should probably support any CSV header format we know our tooling will export. Now to figure out a sane way of how to implement that..

Reply:
Its Time for Another Ticket 😎